app/decorators/article_decorator.rb

```diff
@@ -70,4 +70,14 @@ def description_and_tags
 
     modified_description + " Tagged with #{cached_tag_list}."
   end
+
+  def video_metadata
```

Comment:
**suggestion (non-blocking):** A potential alternative with less repetition:

```ruby
def video_metadata
  as_json(only: [:id, :video_code, :video_source_url, :video_thumbnail_url, :video_closed_caption_track_url])
end
```

In action (I didn't have an article with the right attributes in my local DB):

```ruby
d = ArticleDecorator.new(a)
d.video_metadata
#=>
{
                              "id" => 52,
  "video_closed_caption_track_url" => nil,
                      "video_code" => nil,
                "video_source_url" => nil,
             "video_thumbnail_url" => nil
}
```

Reply:
I like the cleaner look, but is there a way to associate `video_thumbnail_url` to `cloudinary_video_url` method using this approach? Because we're not rendering the exact value of `video_thumbnail_url`, we are using the `cloudinary_video_url` method of the Article.

Maybe another enhancement to be added in the follow up PR

Reply:
@fdoxyz Oh, did I miss that one attribute gets renamed?  

In that case:

```ruby
hash = as_json(..., :cloudinary_video_url)
hash[:video_thumbnail_url] = hash.delete(:cloudinary_video_url)
hash

# or shorter
as_json(..., :cloudinary_video_url).tap |hash|
  hash[:video_thumbnail_url] = hash.delete(:cloudinary_video_url)
end
```